packages/qwik/src/core/use/use-context.ts

```diff
@@ -130,11 +129,11 @@ export interface ContextId<STATE> {
  * @public
  */
 // </docs>
-export const createContextId = <STATE = unknown>(name: string): ContextId<STATE> => {
+export const createContextId = <STATE = unknown>(name: string) => {
```

Comment:
Please keep the annotation here, so that later code changes don't accidentally change the return signature